libs/wingsdk/src/target-tf-aws/app.ts

```diff
@@ -344,4 +358,85 @@ export class App extends CdktfApp {
     this.subnets.private.push(privateSubnet2);
     return this._vpc;
   }
+
+  /**
+   * The ECR Repository for the App
+   */
+  public get ecr(): EcrRepository {
+    if (this._ecr) {
+      return this._ecr;
+    }
+
+    const ecr = new EcrRepository(this, "Ecr", {
+      name: "my-ecr-repo", // TODO: make this configurable
```

Comment:
Let's base the default on the root ID of the app (`{root.id}-images`).

And do this: https://github.com/winglang/wing/issues/6709

Reply:
Yupp, 😁 Im working my way through cleaning up PRs and removing my hard coded "make it work" code